docs/polkadot/implementors-guide/guide.md

```diff
@@ -0,0 +1,342 @@
+|              |      |                  |----> Subprocess A     |    
+| Block Import |      |                  |    +------------------+    
+|    Events    |------+   Overseer       |    +------------------+    
++--------------+      |                  |----> Subprocess B     |    
+                      |                  |    +------------------+    
++--------------+      |                  |    +------------------+    
+|              |      |   Process        |----> Subprocess C     |    
+| Finalization |------+                  |    +------------------+    
+|    Events    |      |                  |---->------------------+    
++--------------+      +------------------+    | Subprocess D     |    
+                                              +------------------+    
+```
+
+[TODO: specific conditions under which new processes are created]
+---
+
+### Candidate Backing Process
+
+The Candidate Backing Process is the process that a validator engages in to contribute to the backing of parachain candidates submitted by other validators.
+
+Many instances of this process may be live at the same time. Each instance is scoped to a particular relay-chain block, known contextually as the _relay parent_.
+
+The goal of the Candidate Backing Process is to produce as many backed candidates as possible. This is done via signed [Statements](#Statement-type) by validators. If a candidate receives a majority of supporting Statements from the Parachain Validators currently assigned, then that candidate is considered backed.
+
+The Candidate Backing Process is straightforward and consists of 3 tasks:
+* Fetching new Statements about candidates from a network interface
+* Performing work based on those Statements. Either validating new candidates that the process was previously unaware of, keeping track of which candidates have received backing, or submitting misbehavior reports.
+* Issuing new Statements based on work done.
+
+In order to carry out its operations, this process depends on input parameters. In practice, this will be fetched from the state of the relay parent using Runtime APIs.
+
+**Parameters**
+* The current Validator set.
+* Validator -> Parachain Assignments
+* Local Key - the current node' validation key, if any.
+* Local Assignment - the parachain the current node is assigned to, as a validator.
+* Local Assignment parachain head - what to treat as the head of the parachain
+* Local Assignment parachain code - the validation function of the parachain
+* Network - a mockable interface for receiving and submitting signed statements about candidates.
+
+If the local key is not present or there is no local assignment, the candidate process will run in an observer mode where it will witness candidates that other validators backs without participating in the process itself.
+
+**Operation**
+
+Pseudocode:
+```rust
+loop {
+  let signed: SignedStatement = network_statements.next()?;
```

Comment:
```suggestion
  let signed: SignedStatement = network_statements.next().await?;
```

While hopefully all logic within this realm can be synchronous, communication to the outside world (or to the lower Substrate levels) will need to be asynchronous.